packages/react/src/primitives/Message/useMessageContext.ts

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { MessageColorTheme } from '../types';
+
+export interface MessageContextProps {
+  colorTheme?: MessageColorTheme;
+  dismissed?: boolean;
+  setDismissed: (dismissed: boolean) => void;
+}
+
+export const MessageContext = React.createContext<MessageContextProps>({
+  dismissed: false,
+  setDismissed: () => {},
+});
+
+export const useMessageContext = (): MessageContextProps =>
```

Comment:
Prefer the more concise `useMessage `

Reply:
Thanks updated in latest to `useMessage`